MailCore/Cache/MailboxManager/MailboxManager+Draft.swift

```diff
@@ -72,6 +77,11 @@ public extension MailboxManager {
                 }
             }
         } catch let error as MailApiError {
```

Comment:
Maybe we can even put it in an additional catch close ?

Reply:
Sure I can do a `catch where`. I had in mind to catch all `MailApiError` in the same scope.

Reply:
As our errors are not currently `enums` but `objects`, instances of `MailApiError` here.

The syntax is less aestetic, but this does __not__ seems to work 
`catch let error as MailApiError where error == MailApiError.apiIdentityNotFound {}`

I can make it work with this ``catch MailApiError.apiIdentityNotFound {}`__if__ I rewrite all MailApiError with an enum.

Reply:
This needs too much refactor let's keep it this way then